commands/init.ts

```diff
@@ -2,6 +2,7 @@ import { Untar } from 'https://deno.land/std@0.100.0/archive/tar.ts'
 import { Buffer } from 'https://deno.land/std@0.100.0/io/buffer.ts'
 import { readAll } from 'https://deno.land/std@0.100.0/io/util.ts'
 import { green, blue, dim, red, cyan } from 'https://deno.land/std@0.100.0/fmt/colors.ts'
+import { validateDenoLand } from "https://deno.land/x/is_valid_package_name/mod.ts";
```

Comment:
- better add the moudle version
- why not use regexp? so we can control the naming rules

Reply:
Yes sure we can use regexp that follow this rules:
- **Name length must be greater than zero**
- **Name cannot start with a period**
- **Name cannot start with an underscore**
- **Name cannot contain leading or trailing spaces**
- **Name can no longer contain more than 214 characters**
- **Name can no longer contain capital letters**
- **Name can no longer contain special characters ("~\'!()*")**